engineering-plans/0002-subgraph-schema-merging.md

```diff
@@ -0,0 +1,118 @@
+# PLAN-0001: Subgraph Schema Merging for Subgraph Composition
```

Comment:
I believe this is `PLAN-0002`. Let's shorten the title also: `Subgraph Schema Merging`.